### DISCUSSION TITLE
wayland support in lira

### DISCUSSION THREAD

**Discussion by Docbroke** [leo-arch/clifm#96] [Ideas]
When file is opened in wayland using clifm. it choose applications from non-GUI `!X` category. In addition to $DISPLAY `clifm` may also check `$WAYLAND_DISPLAY` to decide if GUI environment is running.

**Comment by leo-arch** [comment]
That's really nice info! Never run Wayland myself, so, good to know. I'll add this check immediately so that you can give it a try.

EDIT: Done.

**Reply by Docbroke** [reply]
It is working. I didn't see that as there was no email notification.

**Reply by leo-arch** [reply]
Don't worry. Thanks for the info.
